### PR TITLE
eren-motion-controllers

### PR DESCRIPTION
**Pull request by Ergoold**

### CODE REVIEW

**Comment by Spikes-2212** [comment]
Task linked: [CU-7mq4km add a subPackage to control named speedControllers](https://app.clickup.com/t/7mq4km)

**Comment by Spikes-2212** [comment]
Task linked: [CU-7mq4gd add SmartMotionSettings](https://app.clickup.com/t/7mq4gd)

**Comment by Spikes-2212** [comment]
Task linked: [CU-7mq416 Rename PIDSpeedController to ClosedLoopSpeedController](https://app.clickup.com/t/7mq416)

**Comment by Spikes-2212** [comment]
Task linked: [CU-7mq4bk Add SmartMotionSparkMax](https://app.clickup.com/t/7mq4bk)

File: src/main/java/com/spikes2212/control/SmartMotionSettings.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+import java.util.function.Supplier;
+
+/**
+ * The smart motion constants used in a smart motion loop, in addition to the regular PID constants.
+ */
+public class SmartMotionSettings {
+
+    /**
+     * The minimum velocity of the motor during this smart motion loop.
```
Comment (YuvalLevy1) [inline src/main/java/com/spikes2212/control/SmartMotionSettings.java:11]: change the full sentence into a phrase

**Review by YuvalLevy1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/spikes2212/control/SmartMotionSettings.java (hunk 1 above)
Comment (YuvalLevy1) [reply src/main/java/com/spikes2212/control/SmartMotionSettings.java:11]: This is a short summary, and therefore should consist of full sentences and not a short phrase.

**Pull request merged**